src/chat.rs

```diff
@@ -1190,7 +1171,7 @@ impl Chat {
         }
         schedule_ephemeral_task(context).await;
 
-        Ok(MsgId::new(msg_id))
+        Ok(MsgId::new(u32::try_from(msg_id).unwrap()))
```

Comment:
This `unwrap` looks bad, it will crash the app if negative or too large value appears in the database.